src/main/resources/application.properties

```diff
@@ -1,10 +1,4 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/db
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=1111
 spring.jpa.open-in-view=false
 spring.jpa.hibernate.ddl-auto=validate
```

Comment:
how your application's db should work without these properties? e.g. 
spring.datasource.url=jdbc:mysql://localhost:3306/db
spring.datasource.username=root
spring.datasource.password=1111